packages/core/src/tools/mcp-client.ts

```diff
@@ -301,6 +307,44 @@ export async function discoverTools(
   }
 }
 
+const PromptListResponseSchema = z.object({
```

Comment:
Does the anthropic SDK have this schema somewhere to be re-used by chance? I would have figured they'd have all the types we need